3p/integration.js

```diff
@@ -433,7 +433,8 @@ function triggerRenderStart(opt_data) {
  */
 function observeIntersection(observerCallback) {
   // Send request to received records.
-  if (window.IntersectionObserver && window.IntersectionObserver.prototype) {
+  if (window.IntersectionObserver &&
+      Object.keys(window.IntersectionObserver.prototype).length) {
```

Comment:
We can check for `IntersectionObserver.prototype.observe` instead. Can we add a comment why the test is necessary, too?